src/version-util.ts

```diff
@@ -34,6 +34,8 @@ interface UpgradeOptions {
   removeRange?: boolean
 }
 
+export type UpgradeGroup = 'major' | 'minor' | 'patch' | 'pre-v1' | 'none'
```

Comment:
To follow the project's conventions for type definitions, please move `UpgradeGroup` to its own file at `/src/types/UpgradeGroup.ts`.

Reply:
Done!